.github/workflows/test.yml

```diff
@@ -14,13 +14,9 @@ jobs:
       - uses: actions/checkout@v3
         with:
           submodules: "recursive"
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@v1
         with:
-          profile: minimal
           toolchain: stable
-          override: true
       - name: Cache Rust Dependencies
         uses: Swatinem/rust-cache@v2
-      - uses: actions-rs/cargo@v1
-        with:
-          command: test
+      - run: cargo test --all-targets
```

Comment:
Curious: I wonder the the actual effect of `--all-targets` here is: I thought "tests" is one of the targets? It certainly doesn't matter having it here. I see it is documented in `cargo test --help`: `--all-targets             Test all targets`.